### PR TITLE
[190] Updating docs

### PR DESCRIPTION
**Pull request by paweljakubas**
<p align="right">#190</p>

# Overview

- [x] The API contains instructions about fragmenting a wallet UTxO
- [x] The API contains instructions about creating multi-outputs transactions, describing an ideal and maximum level of fragmentation for exchanges.
- [x] UtxoNotEnoughFragmented added manually to errorsDescription in Swagger

# Comments

### CODE REVIEW

**Comment by paweljakubas** [comment]
adopted stack2nix.patch and squashed

File: src/Cardano/Wallet/API/V1/Swagger.hs [hunk 1]
```diff
@@ -684,6 +684,37 @@ curl -X POST https://localhost:8090/api/v1/transactions \
...
+Hence, the more Utxo one has available, the more transactions one can do at a same time. In essence, by having a more fragmented Utxo, one gains concurrency.
+This is why the proper Utxo fragmentation allows also for making multiple transactions at the same time. For more information, refer [Coin Selection Blog Entry](https://iohk.io/blog/self-organisation-in-coin-selection/)
+
+There's a maximum number of inputs a transaction can have, because transaction have a maximum size (checked by the node). This depends on the initial configuration and, can be voted as part of an update proposal during the blockchain lifecycle. In the case a maximum transaction size is exceeded, `TooBigTransaction` error is triggered.
```
Comment (KtorZ) [inline src/Cardano/Wallet/API/V1/Swagger.hs]: I've made a few adjustments and sent that to Amy for technical review. I'll post her update and adjustments once she's done with it :)

Comment (paweljakubas) [reply src/Cardano/Wallet/API/V1/Swagger.hs]: @KtorZ  please note that stack2nix regeneration is included here and there is separate PR for this

Comment (KtorZ) [reply src/Cardano/Wallet/API/V1/Swagger.hs]: I know, I made the separate PR :smile:

**Review by KtorZ — APPROVED** [review]

**Pull request merged**